Fabric.Identity.API/scripts/Install-IdPSS-Utilities.psm1

```diff
@@ -500,50 +503,53 @@ function Set-IdentityEnvironmentAzureVariables {
     )
-    $scope = "identity"
-	# Alter Identity web.config for azure
-    $clientSettings = Get-ClientSettingsFromInstallConfig -installConfigPath $installConfigPath -appName "Identity Service"
-    $allowedTenants += Get-SettingsFromInstallConfig -installConfigPath $installConfigPath `
-        -scope $scope `
-        -setting "allowedTenants"
-
-    $claimsIssuer += Get-SettingsFromInstallConfig -installConfigPath $installConfigPath `
-        -scope $scope `
-        -setting "claimsIssuerTenant"
-
 	Clear-IdentityEnvironmentAzureSettings -environmentSettingPath $appConfig\web.config
     $environmentVariables = @{}
 
-	# Set Azure Settings
-    $environmentVariables.Add("AzureActiveDirectorySettings_Authority", "https://login.microsoftonline.com/common")
-    $environmentVariables.Add("AzureActiveDirectorySettings_DisplayName", "Azure AD")
-	$environmentVariables.Add("AzureActiveDirectorySettings_ClaimsIssuer", "https://login.microsoftonline.com/" + $claimsIssuer)
-    $environmentVariables.Add("AzureActiveDirectorySettings_Scope_0", "openid")
-	$environmentVariables.Add("AzureActiveDirectorySettings_Scope_1", "profile")
-    $environmentVariables.Add("AzureActiveDirectorySettings_ClientId", $clientSettings.clientId)
-
-    $secret = $clientSettings.clientSecret
-        if($secret -is [string] -and !$secret.StartsWith("!!enc!!:")){
-            $encryptedSecret = Get-EncryptedString  $encryptionCert $secret
-            # Encrypt secret in install.config if not encrypted
-            Add-InstallationTenantSettings -configSection "identity" `
-                -tenantId $clientSettings.tenantId `
-                -clientSecret $encryptedSecret `
-                -clientId $clientSettings.clientId `
-                -installConfigPath $installConfigPath `
-                -appName "Identity Service"
-
-            $environmentVariables.Add("AzureActiveDirectorySettings_ClientSecret", $encryptedSecret)
-        }
-        else{
-            $environmentVariables.Add("AzureActiveDirectorySettings_ClientSecret", $secret)
-        }
-
-    foreach($allowedTenant in $allowedTenants)
-    {
-      $index = $allowedTenants.IndexOf($allowedTenant)
-      $environmentVariables.Add("AzureActiveDirectorySettings_IssuerWhiteList_$index", "https://sts.windows.net/" + $allowedTenant)
-    }
+	if($useAzure -eq $true)
+	{
+		$scope = "identity"
+		# Alter Identity web.config for azure
+		$clientSettings = Get-ClientSettingsFromInstallConfig -installConfigPath $installConfigPath -appName "Identity Service"
+		$allowedTenants += Get-SettingsFromInstallConfig -installConfigPath $installConfigPath `
+			-scope $scope `
+			-setting "allowedTenants"
+
+		$claimsIssuer += Get-SettingsFromInstallConfig -installConfigPath $installConfigPath `
+			-scope $scope `
+			-setting "claimsIssuerTenant"
+
+		# Set Azure Settings
+		$environmentVariables.Add("AzureActiveDirectorySettings_Authority", "https://login.microsoftonline.com/common")
```

Comment:
The separator for these requires a double underscore, for example: `AzureActiveDirectorySettings_Authority` or `AzureActiveDirectorySettings_Scope_0`.

Reply:
Good catch. Thanks.